tests/integration/navigation/sequence/sequence_core.js

```diff
@@ -51,7 +51,7 @@ asyncTest( "Navigating to an internal page", function() {
 			$( "#openInternalPage" ).click();
 		},
 		{
-			pagecontainerchange: { src: $.mobile.pageContainer, event: "pagecontainerchange" + eventNs + "1" }
+			pagecontainerchange: { src: $( ".ui-pagecontainer" ), event: "pagecontainerchange" + eventNs + "1" }
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?